core/src/main/java/org/infinispan/reactive/publisher/impl/LocalPublisherManagerImpl.java

```diff
@@ -83,12 +94,6 @@
    protected int maxSegment;
    protected final int cpuCount = ProcessorInfo.availableProcessors();
 
-   protected CacheSet<K> keySet;
```

Comment:
I wonder if not caching these has caused a performance regression. I know originally I found going down the interceptor stack for every invocation had a measurable cost.

Reply:
The same thing happens when you call the `localPublisher()` method: every cache set wrapper that an interceptor added now wraps the publisher (usually more than once). Plus I removed 1 interceptor, so it should be faster now ;)